src/components/NavBar.js

```diff
@@ -2,23 +2,62 @@ import { NavLink as RouterLink } from 'react-router-dom';
 
 export default function NavBar(props) {
   return (
-    <nav>
+    <nav className="grid grid-cols-2 bg-blue-ncs p-5 text-lg md:text-xl w-full divide-x-2 divide-gray-200 absolute bottom-0 md:px-36 lg:px-48 xl:px-72 lg:w-2/3">
```

Comment:
The nav is a different shade of blue than the Figma, but I think it sticks out more the way you have it here without being too much. All that to say, I like it.

Reply:
That's interesting! The color code from [our palette](https://coolors.co/ef476f-ffd166-06d6a0-118ab2-073b4c) is `#118AB2` which matches the hex code in our `tailwind.config.js` file:

```javascript
 colors: {
  'paradise-pink': '#EF476F',
  'orange-yellow': '#FFD166',
  'caribbean-green': '#06D6A0',
  'blue-ncs': '#118AB2', // same as the color palette hex code
  'midnight-green': '#073B4C',
},
```

But Figma does appear to represent it a bit differently. Weird!

Reply:
I checked and it looks like in Figma it is the same color being used, but a transparency was also used in layering with the background color.